Libplanet/Blockchain/Policies/IBlockPolicy.cs

```diff
@@ -15,6 +17,14 @@ namespace Libplanet.Blockchain.Policies
     public interface IBlockPolicy<T>
         where T : IAction, new()
     {
+        /// <summary>
+        /// A comparer to determine which branch is the canonical chain (i.e., best chain).
+        /// The most greater one according to this comparer is considered to be the canon.
+        /// </summary>
+        /// <seealso cref="IBlockExcerpt"/>
+        /// <seealso cref="Blockchain.CanonicalChainComparer"/>
+        IComparer<IBlockExcerpt> CanonicalChainComparer { get; }
```

Comment:
IMO chain comparer is not really a property related to `Block<T>` but `BlockChain<T>`. Maybe we need something like `IBlockChainPolicy`? Not a strong suggestion though.

Reply:
I believe `IBlockPolicy<T>` is already about `BlockChain<T>`, not `Block<T>`… its namespace implies that too: `Libplanet.Blockchain`.  It might be better to rename to `IBlockChainPolicy<T>` or `IConsensus<T>`? /cc @planetarium/libplanet

Reply:
I think `IBlockPolicy<T>` could be confusing enough as @limebell said. `IConsensus<T>` is much better name for describe.